tools/manifestfile.py

```diff
@@ -603,6 +603,9 @@ def main():
         default=os.path.join(os.path.dirname(__file__), "../lib/micropython-lib"),
         help="path to micropython-lib repo",
     )
+    cmd_parser.add_argument(
+        "--unix-ffi", action="store_true", help="Add dependency to unix-ffi libraries."
```

Comment:
Maybe "prepend unix-ffi to the library path".

Reply:
Done!